public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -11,6 +11,11 @@ import { getScaledDimension, } from '../../dims/scale';
 import { getColorDimension, } from '../../dims/color';
 import { ScaleDimensionEditor } from '../../dims/editors/ScaleDimensionEditor';
 import { ColorDimensionEditor } from '../../dims/editors/ColorDimensionEditor';
+import React from 'react';
```

Comment:
```suggestion
import React { ReactNode } from 'react';
```
usually we have react as the first import -- not sure why :man_shrugging: